service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/azure/container/DestinationStorageAccount.java

```diff
@@ -0,0 +1,16 @@
+package bio.terra.workspace.service.resource.controlled.flight.clone.azure.container;
+
+import java.util.UUID;
+import javax.annotation.Nullable;
+
+/**
+ * Represents a destination storage account for the purposes of cloning.
```

Comment:
This is an attempt at capturing an `Either` semantic for the destination storage account (i.e,. it's either an LZ storage account or a workspace storage account).